VimeoNetworking/Sources/ResponseCache.swift

```diff
@@ -40,7 +40,7 @@ private protocol Cache
 }
 
 /// Response cache handles the storage of JSON response dictionaries indexed by their associated `Request`.  It contains both memory and disk caching functionality
-final internal class ResponseCache
+final public class ResponseCache
```

Comment:
is it intentional to make this guy public?

Reply:
Yeaaah, since it was internally accessible to the module, but not accessible to the test targets which live outside of that.

Reply:
@jasonhawkins Isn't that why you would use `@testable import` rather than `import` in the test files?

Reply:
https://www.natashatherobot.com/swift-2-xcode-7-unit-testing-access/

Reply:
I know how this is _supposed_ to work, but if this class isn't public it won't be accessible from the test target. Even though it should be. The `@testable` attribute is supposed to allow us do exactly that, but it's not. Here's Apple's documentation:

> When you write an app with a unit test target, the code in your app needs to be made available to that module in order to be tested. By default, only entities marked as open or public are accessible to other modules. However, a unit test target can access any internal entity, if you mark the import declaration for a product module with the @testable attribute and compile that product module with testing enabled.

For our part we're using `@testable` for the import declaration and the product module does have testing enabled. 

I looked into this and created a new example class and example test case. I tried out the different access levels and ran into the exact same issue. Even though I was using `@testable`, internal classes were not accessible from the test case. BUT! I have a solution…

As of CocoaPods 1.3.0 we can move all of our tests _directly into the pod_. This seems like a pretty good option. I've got a WIP branch that demonstrates what that world might look like, built with CocoaPods 1.5.0: https://github.com/vimeo/VimeoNetworking/tree/tech/swift-4-cocoapods-1-5-0. And I've got a document with some of the high level changes between CocoaPods 1.3.1, 1.4.0, and 1.5.0 here: https://paper.dropbox.com/doc/CocoaPods-1.3.1-1.4.0-1.5.0-vDVpGmtp4IJKFoFtia9Ii.

What I'd like to do is either disable the tests that fail (temporarily) because of this access control issue, or change the access level of relevant classes (again, temporarily) for this PR and then work on getting us up-to-date with CocoaPods 1.5.0 and merge those changes too.